src/backend/base/langflow/components/logic/conditional_router.py

```diff
@@ -27,16 +29,17 @@ def __init__(self, *args, **kwargs):
         DropdownInput(
             name="operator",
             display_name="Operator",
-            options=["equals", "not equals", "contains", "starts with", "ends with"],
+            options=["equals", "not equals", "contains", "starts with", "ends with", "matches regex"],
             info="The operator to apply for comparing the texts.",
             value="equals",
+            real_time_refresh=True,
         ),
         BoolInput(
             name="case_sensitive",
             display_name="Case Sensitive",
             info="If true, the comparison will be case sensitive.",
             value=False,
-            advanced=True,
+            advanced=False,
```

Comment:
this is the default right? I think we can remove this line.

Reply:
I agree!